core-bundle/contao/widgets/MetaWizard.php

```diff
@@ -171,7 +171,7 @@ public function generate()
 			// Add the input fields
 			foreach ($this->varValue as $lang=>$meta)
 			{
-				$item = '<li data-language="' . $lang . '"><span class="lang">' . ($languages[$lang] ?? $lang) . ' ' . Image::getHtml('delete.svg', '', 'class="tl_metawizard_img" title="' . $GLOBALS['TL_LANG']['MSC']['delete'] . '" data-delete') . '</span>';
+				$item = '<li data-language="' . $lang . '" data-controller="metawizard"><span class="lang">' . ($languages[$lang] ?? $lang) . ' <button data-action="metawizard#delete:prevent">' . Image::getHtml('delete.svg', '', 'class="tl_metawizard_img" title="' . $GLOBALS['TL_LANG']['MSC']['delete'] . '"') . '</button></span>';
```

Comment:
Just a note to myself: adding the `button` element needs to be backported to Contao 4 to fix the accessibility of the function.

Reply:
see https://github.com/contao/contao/pull/5411